pkg/chain/ethereum_chain.go

```diff
@@ -27,14 +28,16 @@ type ETHChain struct {
 
 // NewRequest : Takes signed transaction data as a parameter
```

Comment:
This comment is no longer relevant. Change it to mention that is instead returns a `PostRequest` struct

Reply:
In fact, this comment should be in the `IChain` interface definition.

Reply:
Thanks.